demos/segmentation_demo/README.md

```diff
@@ -17,36 +15,37 @@ Inference Engine plugin. When inference is done, the application creates an outp
 Running the application with the `-h` option yields the following usage message:
 ```sh
 ./segmentation_demo -h
-InferenceEngine:
-    API version ............ <version>
-    Build .................. <number>
+[ INFO ] InferenceEngine: <version>
+[ INFO ] Parsing input parameters
 
 segmentation_demo [OPTION]
 Options:
 
     -h                        Print a usage message.
-    -i "<path>"               Required. Path to a .bmp image.
+    -i "<path>"               Required. Input to process.
     -m "<path>"               Required. Path to an .xml file with a trained model.
-      -l "<absolute_path>"    Required for CPU custom layers. Absolute path to a shared library with the kernel implementations.
+      -l "<absolute_path>"    Required for CPU custom layers. Absolute path to a shared library with the kernels implementations.
           Or
-      -c "<absolute_path>"    Required for GPU custom kernels. Absolute path to the xml file with the kernel descriptions.
+      -c "<absolute_path>"    Required for GPU custom kernels. Absolute path to the .xml file with the kernels descriptions.
     -d "<device>"             Optional. Specify the target device to infer on (the list of available devices is shown below). Default value is CPU. Use "-d HETERO:<comma-separated_devices_list>" format to specify HETERO plugin. The demo will look for a suitable plugin for a specified device.
+    -no_show                  Optional. Do not visualize inference results.
+    -u                        Optional. List of monitors to show initially.
 ```
 
-Running the application with the empty list of options yields the usage message given above and an error message.
+Running the application with the empty list of options yields an error message.
 
-To run the demo, you can use public or pre-trained models. To download the pre-trained models, use the OpenVINO [Model Downloader](../../tools/downloader/README.md) or go to [https://download.01.org/opencv/](https://download.01.org/opencv/).
+To run the demo, you can use public or pre-trained models. To download the pre-trained models, use the OpenVINO [Model Downloader](../../tools/downloader/README.md) or go to [https://download.01.org/opencv/](https://download.01.org/opencv/). The list of pre-trained models supported by the demo is in [models.lst](./models.lst).
```

Comment:
```suggestion
To run the demo, you can use public or pre-trained models. To download the pre-trained models, use the OpenVINO [Model Downloader](../../tools/downloader/README.md) or go to [https://download.01.org/opencv/](https://download.01.org/opencv/). The list of models supported by the demo is in [models.lst](./models.lst).
```

Given that you've added `deeplabv3` to the list, I don't think "pre-trained" is appropriate.

Reply:
I will accept it to keep consistency. But I've never thought about it until now: this is strange to have formulations like `public or pre-trained models`. It should be `public or Intel models`. `deeplabv3` is still pre-trained, just not by Intel.

Reply:
I agree, we need to revamp the terminology at some point.